prism-agent/service/server/src/main/scala/io/iohk/atala/issue/controller/http/CreateIssueCredentialRecordRequest.scala

```diff
@@ -39,6 +39,9 @@ final case class CreateIssueCredentialRecordRequest(
     @description(annotations.credentialFormat.description)
     @encodedExample(annotations.credentialFormat.example)
     credentialFormat: Option[String],
+    @description(annotations.isRevocable.description)
+    @encodedExample(annotations.isRevocable.example)
+    isRevocable: Option[Boolean],
```

Comment:
@shotexa So it was decided to finally include that field as part of the request and not to make all JWT credentials revocable by default?

Reply:
Yep, I wouldn't add it if it's not needed for JWT.